### DISCUSSION TITLE
Suggestion: syndicate toy vending machine dispences real item with small chance

### DISCUSSION THREAD

**Discussion by Fogapod** [unitystation/unitystation#7517] [Ideas]
### Description
The idea is that toy vending machine can dispense real counterpart of toy item (sword/gun/blade/armor).
Chance should not be too high to not be abusable, say 1%.

Basic implementation would require adding support of defining alternative drop with configurable chance for vending machines. Maybe this could be extended to a more general system with more flexible configuration, but I don't know how this could be extended yet.

Related vending machine prefab:
https://github.com/unitystation/unitystation/blob/eccc2943f030014f497e66923ad982c8302d0f1a/UnityProject/Assets/Resources/Prefabs/Objects/Machines/VendingMachines/SyndicateDonksoftToyVendor.prefab

As a bonus, if there is/will be syndicate vending machine with equipment, it could dispense toy items with 1% chance instead. Wouldn't this be funny if they won't notice toy gear until guns start firing?

**Comment by PerfectTangent** [comment]
I approve of this idea if and when vending machines actually charge money. Then it's not quite as easily exploited.
